### PR TITLE
Documentation restructured

### PR DESCRIPTION
**Pull request by greatfireball**

### CODE REVIEW

File: doc/alitv-filter.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# SYNOPSIS
+
+    # Filter input file based on settings in JSON
+    alitv-filter.pl --input in.json --output out.json
+
+    # Filter input file based on settings in JSON in combination with
+    # the command line settings
+    alitv-filter.pl --input in.json --output out2.json --min-link-length 10000 --min-link-identity 90%
+
+    # Filter input file based the command line settings and ignoring
```
Comment (iimog) [inline doc/alitv-filter.md]: typo: based -> "based on"

File: doc/alitv-filter.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+# DESCRIPTION
+
+The script filters AliTV JSON files for postprocessing.
+
+# OPTIONS
+
+- `--help|?|man`  Help
+
+    Shows description of this programs, its usage, and its parameters/options.
```
Comment (iimog) [inline doc/alitv-filter.md]: typo: programs -> program

File: doc/alitv-filter.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- --input  Input file
+
+    The name of the input file. Default value is input from STDIN which
+    might be switched on by `--input -`.
```
Comment (iimog) [inline doc/alitv-filter.md]: What do you mean with "might be switched on"? If it is the default it does not need to be switched on, right? Maybe rather: "[...] which is equivalent to `--input -`"

File: doc/alitv-filter.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+- --output   Output file
+
+    The name of the output file. Default value is output to STDOUT which
+    might be switched on by `--output -`.
```
Comment (iimog) [inline doc/alitv-filter.md]: See comment above

File: doc/alitv-filter.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    of a percentage value is assumed.
+
+- `--min-link-length`/`--max-link-length`   Minimum/Maximum length for Links
+
+    Specifies the minimal or maximum length for links. A integer is
+    expected and specifies the length in nucleotids.
+
+- `--min-seq-length`/`--max-seq-length`   Minimum/Maximum length for chromosomes
+
+    Specifies the minimal or maximum length for chromosomes. A integer is
```
Comment (iimog) [inline doc/alitv-filter.md]: typo: "A integer" -> "An integer"

File: doc/alitv-filter.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    expected and specifies the length in nucleotids.
+
+- `--ignore-json` Ignore settings in JSON file
+
+    Speciefies if the current settings from the JSON should be ignored,
```
Comment (iimog) [inline doc/alitv-filter.md]: if -> that ?

**Review by iimog — CHANGES_REQUESTED** [review]
I like the new structure, just some minor remarks.

File: doc/alitv-filter.md (hunk 1 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: done

File: doc/alitv-filter.md (hunk 2 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: check

File: doc/alitv-filter.md (hunk 3 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: Rephrased: "Default value is input from STDIN which can be manually enabled by --input -"

File: doc/alitv-filter.md (hunk 4 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: dito

File: doc/alitv-filter.md (hunk 5 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: check

File: doc/alitv-filter.md (hunk 6 above)
Comment (greatfireball) [reply doc/alitv-filter.md]: Rephrased: "If enabled, the current settings from the JSON are ignored,[...]"

**Review by iimog — APPROVED** [review]

**Pull request merged**